packages/components/src/components/Log/Log.js

```diff
@@ -122,6 +145,9 @@ export class LogContainer extends Component {
             })
           ]
         });
+        if (stepStatus && !stepStatus.terminated) {
+          this.timer = setTimeout(() => this.loadLog(), pollingInterval);
```

Comment:
no need for the additional anonymous function here

```suggestion
          this.timer = setTimeout(this.loadLog, pollingInterval);
```